spec/models/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    before(:example) { get user_posts_path(1) }
+
+    it 'should have correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should include correct placeholder text' do
+      expect(response.body).to include('Posts#index')
+    end
+  end
```

Comment:
- [ ] This should make your tests not run because we are supposed to target the text that has been included in the body. however, you included ('Posts#index).

HINT: expect(response.body).to include("Welcome to user index page").

Reply:
if you chick the`index.html.erb` you will find that `Posts#index` is inside the h1 it's the same thing

Reply:
I know you will find it but we need to check if the body includes that text.